framework-plugins/lisk-framework-http-api-plugin/swagger.yml

```diff
@@ -66,3 +47,768 @@ info:
 
 # All paths relative to specified basePath
 basePath: /api
+
+# Tags for organizing operations
+tags:
+  - name: Accounts
+    description: Account related API endpoints.
+  - name: Blocks
+    description: Block related API endpoints.
+  - name: Delegates
+    description: Delegates related API endpoints.
+  - name: Node
+    description: Node related API endpoints.
+  - name: Peers
+    description: Peers related API endpoints.
+  - name: Transactions
+    description: Transactions related API endpoints.
+
+schemes:
+  - http
+
+paths:
+  /accounts/{address}:
+    get:
+      tags:
+        - Accounts
+      summary: Requests account data.
+      description: Search for matching account in the system.
+      produces:
+        - application/json
+      parameters:
+        - name: address
+          in: path
+          example: nQFJsJYtRL/Aip9k1a/Otigdf7U=
+          description: |
+            The Lisk address is the human readable representation of the accounts owners public key in base64 format.
+          required: true
+          schema:
+            type: string
+      responses:
+        200:
+          description: Account requested.
+          schema:
+            $ref: '#/definitions/AccountResponse'
+        400:
+          description: Malformed query or parameters.
+          schema:
+            $ref: '#/definitions/ErrorResponse'
+        404:
+          description: Data not found.
+          schema:
+            $ref: '#/definitions/ErrorResponse'
+        429:
+          description: Too many requests, exceeded rate limit.
+          schema:
+            $ref: '#/definitions/ErrorResponse'
+        500:
+          description: Unexpected error.
+          schema:
+            $ref: '#/definitions/ErrorResponse'
+  /delegates:
+    get:
+      tags:
+        - Delegates
+      summary: Requests delegates data.
+      description: Search for a specified delegate in the system.
+      produces:
+        - application/json
+      parameters:
+        - $ref: '#/parameters/limit'
+        - $ref: '#/parameters/offset'
+      responses:
+        200:
+          description: Registered delegate accounts.
+          schema:
+            $ref: '#/definitions/DelegateResponse'
```

Comment:
It's missing the pagination information. With given `limit`, `offset` how will one knows he needs to paginate further results.